tests/2015-04-01/Microsoft.DomainRegistration.tests.json

```diff
@@ -0,0 +1,81 @@
+{
+  "tests": [
+    {
+      "name": "domains - basic",
+      "definition": "http://schema.management.azure.com/schemas/2015-04-01/Microsoft.DomainRegistration.json#/resourceDefinitions/domains",
+      "json": {
+        "apiVersion": "2015-04-01",
+        "name": "validDomain",
+        "type": "Microsoft.DomainRegistration/domains",
+        "properties": {
+          "contactAdmin": {
+                            "email": "abc@live.com", 
```

Comment:
Can you please format the files so that they follow the json style in this repo?